src/Symfony/Component/Config/Definition/Builder/ArrayNodeDefinition.php

```diff
@@ -80,6 +80,76 @@ public function prototype($type)
     }
 
     /**
+     * Shorthand for prototype('variable').
+     *
+     * @return VariableNodeDefinition
+     */
+    public function prototypeVariable()
+    {
+        return $this->prototype('variable');
+    }
+
+    /**
+     * Shorthand for prototype('scalar').
+     *
+     * @return ScalarNodeDefinition
+     */
+    public function prototypeScalar()
```

Comment:
I agree, my IDE does that too, I was only thinking about some friends who are still using IDEs with left prefix only autocomplete support (Netbeans 6). If you think we shouldn't care about this legacy, I am into it.